ecosystem/python/sdk/README.md

```diff
@@ -18,16 +18,26 @@ Currently this is still in development and is unsuitable for directly interfacin
 
 ### Testing
 
+To run all unit tests:
 ```bash
 python3 -m unittest discover -s aptos_sdk/ -p '*.py'
 ```
 
+To run all end-to-end tests:
+```bash
+ln -s `pwd`/aptos_sdk examples/
+python3 aptos_sdk/coin.py
```

Comment:
Okay so we're only supporting python3 and being explicit?

Reply:
well it was always only python3 and probably a pretty recent version... we'll leave this for figuring out later..